README.md

```diff
@@ -52,11 +52,24 @@ The package has been tested in the following configuration:
 Use composer to install the package:
 
 ```
+composer require laravel/scout
```

Comment:
Actually, `scout-elasticsearch-driver` requires `laravel/scout` in its composer.json, so no need to require it manually.